lms/templates/_gated_content.html

```diff
@@ -0,0 +1,26 @@
+<%page args="prereq_url, prereq_section_name, gated_section_name" expression_filter="h"/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import Text
+%>
+
+<div class="xblock xblock-student_view xblock-student_view-vertical xblock-initialized">
```

Comment:
This ```div``` needs a few more attributes to make it work with the tab panel markup in the ```nav``` ```id="seq_content"```  and ```role="tabpanel"```  and ```aria-labelledby="tab_0"```

Reply:
So I added a separate div beneath this div that the tab pretends to control
```<div id="seq_content" role="tabpanel" aria-labelledby="tab_0"></div>```
This div will correctly be empty as the sequence content is not displayable because it's gated.

If I added the attributes to the div you specified, the "Content Locked" content doesn't get displayed because the javascript is doing some dynamic replacement of id="seq_contents" on page load trying to load the real sequence_content XBlock, but that is gated so it's replace with empty content